src/types/expanded_menu_type.py

```diff
@@ -0,0 +1,24 @@
+from graphene import List, ObjectType, String
+
+from src.types.food_station_type import FoodItemType, FoodStationType
+
+class DescriptiveFoodItemOptionType(ObjectType):
+  label = String(required=True)
+  options = List(String, required=True)
+
+class DescriptiveFoodItemType(FoodItemType):
+  options = List(DescriptiveFoodItemOptionType, required=False)
```

Comment:
same note as in parsing, should keep naming conventions consistent with json file.